content/docs/run-core-node/commands.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: Commands
+order: 80
+---
+import { CodeExample } from "components/CodeExample"
+
+Stellar Core can be controlled via the following commands.
+
+## Common options
+Common options can be placed at any place in the command line.
+
+* **--conf <FILE-NAME>**: Specify a config file to use. You can use '-' and
```

Comment:
I had to wrap ` ` around `<FILE-NAME>`, `<LEVEL>` and etc. gatsby-mdx thought it was a component